app/provider.tsx

```diff
@@ -34,31 +34,46 @@ if (typeof window !== "undefined") {
   (window as any).posthog = posthog;
 }
 
+function isBitgetWalletInstalled() {
+  // Check if the Bitget wallet is available on the window object
+  return typeof window !== "undefined" && window.bitget !== undefined;
+}
+
 export const availableConnectors = [
   new InjectedConnector({ options: { id: "braavos", name: "Braavos" } }),
   new InjectedConnector({ options: { id: "argentX", name: "Argent X" } }),
-  new InjectedConnector({ options: { id: "bitkeep", name: "Bitget Wallet" } }),
-  new InjectedConnector({ options: { id: "okxwallet", name: "Okx Wallet" } }), // Added okxwallet
+  new InjectedConnector({
+    options: {
+      id: "bitkeep",
+      name: isBitgetWalletInstalled()
+        ? "Bitget Wallet"
+        : "Install Bitget Wallet",
```

Comment:
That will work for now but I think we need to completely reorganize the way we're managing wallets following app.starknet.id model